install_config/install/planning.adoc

```diff
@@ -117,7 +117,7 @@ not supported.
 === Single Master and Multiple Nodes
 
 The following table describes an example environment for a single
-xref:../../architecture/infrastructure_components/kubernetes_infrastructure.adoc#master[master] (with embedded *etcd*)
+xref:../../architecture/infrastructure_components/kubernetes_infrastructure.adoc#master[master] (with *etcd* on the same host)
```

Comment:
Note that it doesn't mention installing etcd. That's because it is using the etcd embedded in the openshift binary